frontend/ui/widget/iconwidget.lua

```diff
@@ -0,0 +1,70 @@
+--[[
+Subclass of ImageWidget to show icons
+]]
+
+local DataStorage = require("datastorage")
+local ImageWidget = require("ui/widget/imagewidget")
+local Screen = require("device").screen
+
+-- Directories to look for icons by name, with any of the accepted suffixes
+local ICONS_DIRS = {}
+local user_icons_dir = DataStorage:getSettingsDir() .. "/icons"
```

Comment:
I guess this should be `DataStorage:getDataDir()` instead of `DataStorage:getSettingsDir()`.
(`koreader/icons/` instead of `koreader/settings/icons/`)

Reply:
Yes, it doesn't seem to have much to do with settings.